src/python/CMakeLists.txt

```diff
@@ -24,9 +20,8 @@ make_copy_target(python_source
   DIRECTORIES
     ${CMAKE_CURRENT_SOURCE_DIR}/turicreate
 )
-add_dependencies(python_source numpy_integration tctensorflow)
```

Comment:
is this `numpy_integration` making building python_source slow?

Reply:
No. See my description. That target doesn’t exist any more.

tctensorflow is what’s making it slow.

Reply:
Ah, I see. I was wondering why `make` would block on a non-existent object.